WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1683,7 +1683,11 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
 
 
     public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        tableView.deselectRow(at: indexPath, animated: false)
+        if !UIAccessibilityIsReduceMotionEnabled() {
+            // This can conflict with the reduce motion fullscreen push animation
+            tableView.deselectRow(at: indexPath, animated: false)
```

Comment:
Thanks for the suggestion – I've moved `deselectRow` to after `pushFullscreenViewController`.